source/features/prevent-link-loss.tsx

```diff
@@ -17,19 +17,27 @@ import {
 } from '../github-helpers/prevent-link-loss.js';
 import createBanner from '../github-helpers/banner.js';
 
+const fieldSelector = [
+	'textarea.js-comment-field',
+	'textarea[aria-labelledby="comment-composer-heading"]', // Commit input field
+];
```

Comment:
I think if you this you won't need `as HTMLTextAreaElement` later
```suggestion
] as const;
```

Reply:
If we use `as const`, this type may not be `strings[]`

Reply:
https://github.com/refined-github/refined-github/blob/6c3e648a8fe312f5c45da960ecc025a403a2ff49/source/github-events/on-commit-title-update.ts#L3-L6

Here hasn't use it also.

Reply:
It does now :) https://github.com/fregante/delegate-it/releases/tag/v6.2.1